test/util/util.go

```diff
@@ -0,0 +1,202 @@
+package util
+
+import (
+	"context"
+	"os"
+	"os/exec"
+
+	ksapi "github.com/kubestellar/kubestellar/api/control/v1alpha1"
+	ksClient "github.com/kubestellar/kubestellar/pkg/generated/clientset/versioned"
+	"github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	ocmWorkClient "open-cluster-management.io/api/client/work/clientset/versioned"
+)
+
+func GetConfig(context string) *rest.Config {
+	config, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		&clientcmd.ClientConfigLoadingRules{ExplicitPath: os.Getenv("KUBECONFIG")},
+		&clientcmd.ConfigOverrides{CurrentContext: context}).ClientConfig()
+	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
+	return config
+}
+
+func CreateClient(config *rest.Config) *kubernetes.Clientset {
```

Comment:
FYI: I would add something to the name of this func, such as "K8s", indicating which kind of client it is making.

Reply:
Done